### ISSUE TITLE
NPC: Magatha Grimtotem

### ISSUE THREAD

**Issue by underlord-oberon** [plantain-gnocchis/wow-voiceover-tbc-wotlk#70]
Add-on version: AI_VoiceOver_v1.4.5
Data version: AI_VoiceOverData_VanillaExtra_v1.4

Thunder Bluff / Elder Rise / NPC: Magatha Grimtotem
Link: https://www.wowhead.com/wotlk/npc=4046/magatha-grimtotem

Greeting: despite the voice-over is perfect, [Magatha voice](https://www.wowhead.com/wotlk/npc=4046/magatha-grimtotem#sounds) is a bit different from the default female tauren voice.

Quests: the same occurs on the quests Magatha [starts](https://www.wowhead.com/wotlk/npc=4046/magatha-grimtotem#starts) or [ends](https://www.wowhead.com/wotlk/npc=4046/magatha-grimtotem#ends).

**Comment by celguar** [comment]
How do these sound?
https://github.com/celguar/wow-voiceover/releases/download/fixes/magatha_quests.zip

**Comment by underlord-oberon** [comment]
> How do these sound?

Looks wonderful. It transmits a real sense of immersion on the story.

**Comment by carevass** [comment]
This should be fixed with new audio in AI_VoiceOverData_VanillaExtra_v1.5.
